code/obj/item/misc_weapons.dm

```diff
@@ -711,6 +732,11 @@ TYPEINFO(/obj/item/sword/pink/angel)
 	stamina_cost = 20
 	stamina_crit_chance = 60
 	// pickup_sfx = 'sound/items/blade_pull.ogg'
+	gang
+		force = 12
+		throwforce = 6
+		stamina_damage = 40
+		stamina_crit_chance = 20
```

Comment:
Nunchucks are kind of meant to be a stamina weapon, if you don't want gangs having stamina weapons then maybe just don't give them nunchucks?